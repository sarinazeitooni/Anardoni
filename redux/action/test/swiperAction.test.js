import React from 'react';
import { shallow } from 'enzyme';
import swiperAction from "../Swiperaction";
describe(" swiperAction " , ()=>{
    it("it should be correct",()=>{
        const container = shallow(<swiperAction/>)
        expect(container).toMatchSnapshot();
    })
});