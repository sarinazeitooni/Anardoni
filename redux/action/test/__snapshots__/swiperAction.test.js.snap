// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` swiperAction  it should be correct 1`] = `ShallowWrapper {}`;
